Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs

```diff
@@ -15,6 +15,24 @@ namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Synchronization;
 [TestClass]
 public class VerificationStatusTest : ClientBasedLanguageServerTest {
 
+  [TestMethod]
+  public async Task ManuallyRunMethodWithTwoUnderlyingTasks() {
+    var source = @"
+method Foo() returns (x: int) ensures x / 2 == 1; {
+  return 2;
+}";
+    await SetUp(new Dictionary<string, string> {
+      { $"{DocumentOptions.Section}:{nameof(DocumentOptions.Verify)}", nameof(AutoVerification.Never) }
+    });
+    var documentItem = CreateTestDocument(source);
+    await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
+    var fooSymbol = (await RequestDocumentSymbol(documentItem)).Single();
+    var fooPosition = fooSymbol.SelectionRange.Start;
+    var runSuccess = await client.RunSymbolVerification(documentItem, fooPosition, CancellationToken);
+    Assert.IsTrue(runSuccess);
+    await WaitForStatus(fooSymbol.SelectionRange, PublishedVerificationStatus.Correct, CancellationToken);
```

Comment:
The two tasks are the well-formedness and the correctness, right?
How does this prove that you are running the second task? What would have it generated as an output before?

Reply:
Before it never published the 'Correct' notification, because it only does that once _all_ underlying tasks have completed.